package.json

```
@@ -7,7 +7,8 @@
   "scripts": {
     "test": "jest --watch --no-cache",
     "start": "webpack serve --open",
-    "build": "webpack"
+    "build": "webpack",
+    "cypress:open": "cypress open"
```

cypress가 추가되었으니 jest를 돌렸을 때 cypress 테스트 케이스가 포함되지 않게 jest Ignore 세팅을 해줍시다 :)

```suggestion
    "cypress:open": "cypress open"
  },
  
  "jest": {
    "testPathIgnorePatterns": [
      "cypress"
    ]
  },
```

---

jest는 기본 폴더만 참조하는 줄 알고 크게 신경안썼는데,

따로 ignore과 같은 설정도 해줄 수 있군요! 감사합니다!